setup.py

```diff
@@ -40,7 +40,7 @@ def local_file(name):
 extras = {
     'datetime':  ["pytz"],
     'fakefactory': ["Faker>=0.7.0,<=0.7.1"],
-    'django': ['pytz', 'django>=1.7'],
+    'django': ['pytz', 'django>=1.9,<2'],
```

Comment:
This omits 1.8.x (supported) and includes 1.9.x (unsupported).  I think the [correct specifier](https://www.python.org/dev/peps/pep-0440/#version-specifiers) would be ` django>=1.8,<2,!=1.9.*`